docs/install/install.rst

```diff
@@ -336,4 +336,63 @@ issues locating the installed libraries:
 .. note::
 
    If ROCm support is enabled, you might have to add the path to the ROCm libraries to ``LD_LIBRARY_PATH``,
-   for example, ``export LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH}``.
\ No newline at end of file
+   for example, ``export LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH}``.
+
+Post-installation troubleshooting
+========================================
+
+This section explains how to resolve certain issues that might happen when you first use Omnitrace.
+
+Issues with RHEL and SELinux
+----------------------------------------------------
+
+RHEL (Red Hat Enterprise Linux) distributions of Linux automatically enable a security feature 
+named SELinux (Security-Enhanced Linux) that prevents Omnitrace from running.
+This issue applies to systems running the CentOS 9 operating system using
+any version of the AMD ROCm Software alongside an AMD Instinct MI300 GPU.
```

Comment:
This can be more generalized to just any distro that has SELinux installed. The RHEL-based distros, like CentOS, Rocky, Alma, Fedora, and RHEL itself, are just examples. 
Likewise, you can see this with any GPU (or none at all)

Reply:
Okay, I'll make this more generic, but will include a couple of examples (RHEL and CentOS are the most popular of these).